src/main/java/mate/academy/springbootintro/controller/BookController.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.springbootintro.controller;
+
+import java.util.List;
+import mate.academy.springbootintro.model.Book;
+import mate.academy.springbootintro.service.BookService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
```

Comment:
add `server.servlet.context-path=/api` to application.properties
```suggestion
@RequestMapping("/books")
```

Reply:
fix comment